lib/services/logs.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+
+import 'package:flutter/widgets.dart';
+import 'package:mobile_nebula/services/result.dart';
+
+class LogsNotFoundException implements Exception {
+  String error() => 'No logs file found';
```

Comment:
I'm not sure that the word "file" is important to the user here, since that's an implementation detail.  I'd suggest instead we say, "No logs found", or if we want to give the user a hint as to why there are no logs, something like, "No logs found.  Has the site been started?"